### PR TITLE
[AITT] Support AITT as edge connection type  @open sesame 9/6 11:16

### PR DESCRIPTION
**Pull request by gichan-jang**
- Support AITT as edge connection type.
 - Add AITT unittest cases

TODO: Support multi memory transfer. (Serialize and Deserialize the data)

Signed-off-by: gichan <gichan2.jang@samsung.com>

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -61,6 +61,19 @@ IF(ENABLE_PAHO_MQTT)
     SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DENABLE_MQTT=1")
 ENDIF()
 
+# AITT Library
+SET(ENABLE_AITT OFF)
```
Comment (myungjoo) [inline CMakeLists.txt:65]: Get user option for AITT.
If user has explicitly enabled AITT and AITTLIB is not found --> error

e.g., https://tttsss77.tistory.com/193

Comment (gichan-jang) [reply CMakeLists.txt]: Thanks. I added options for MQTT and AITT. PTAL.

**Review by myungjoo — APPROVED** [review]
LGTM

File: src/libnnstreamer-edge/nnstreamer-edge-aitt.c [hunk 2]
```diff
@@ -23,11 +28,218 @@ typedef void *nns_edge_aitt_sub_h;
...
+
+  if (nns_edge_data_create (&data_h) != NNS_EDGE_ERROR_NONE) {
+    nns_edge_loge ("Failed to create data handle in msg thread.");
+    return;
+  }
+
+  /** @todo support multi memory chunk. Deserialize the received data. */
+  nns_edge_data_add (data_h, (void *) msg, msg_len, NULL);
+
+  _nns_edge_invoke_event_cb (eh, NNS_EDGE_EVENT_NEW_DATA_RECEIVED, data_h,
```
Comment (jaeyun-jung) [inline src/libnnstreamer-edge/nnstreamer-edge-aitt.c:210]: Q: Does it need destroying data_h after calling event callback? (Now data handle should be copied if it is necessary.)

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**